bats/fb-verify-packages.bats

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:ts=2:et
+
+set -o pipefail
+
+load os_helper
+load foreman_helper
+
+@test "verify no old SCL packages" {
+  run tSetOSVersion
+  run FOREMAN_VERSION=$(tForemanVersion)
+
+  if tIsRedHatCompatible || tIsCentOSCompatible; then
+    if [ $FOREMAN_VERSION == '1.20' ];then
```

Comment:
Would it make sense to write this `[[ $(echo "$FOREMAN_VERSION >= 1.20" | bc) == 1 ]]`

Reply:
What is `bc` ? My environment does not recognize it

Reply:
https://www.gnu.org/software/bc/bc.html

Reply:
Sounds OK for now.